ansible/roles/ocp4-workload-ccnrd/tasks/workload.yml

```diff
@@ -25,6 +25,9 @@
   debug:
     msg: "selected modules list: {{ modules }}"
 
+- name: create ccn CatalogSource
```

Comment:
No `k8s?

Reply:
I'm actually wondering how k8s module enables me to create 2 catalog sources? It would be great if you point me 👍

Reply:
Split in two files?

Reply:
yeah, you read my mind :D I just splitter and tested it in my ocp 4.6.1 cluster.
Can you review it again? thx